actions/setup-conftest/action.yaml

```
@@ -36,7 +44,7 @@ runs:
       with:
         repo: "open-policy-agent/conftest"
         version: "tags/v${{ inputs.version }}"
-        file: "conftest_${{ inputs.version }}_${{ runner.os }}_${{ env.ARCH || runner.arch }}.tar.gz"
+        file: "conftest_${{ inputs.version }}_${{ env.OS || runner.os }}_${{ env.ARCH || runner.arch }}.tar.gz"
```

If you keep it the way you have it, the fallback shouldn't be needed since you unconditionally set it.
```suggestion
        file: "conftest_${{ inputs.version }}_${{ env.OS }}_${{ env.ARCH || runner.arch }}.tar.gz"
```

In other words, pick one or the other of this and the above suggestion 😁